.kokoro/build.sh

```diff
@@ -103,6 +103,30 @@ samples)
         echo "no sample pom.xml found - skipping sample tests"
     fi
     ;;
+tutorials-samples)
```

Comment:
I think build.sh is a file that's generated by owlbot. If you touch it, it'll revert it back. Do we really need this change?

Reply:
I see you've opted out below. What's the reasoning for running this as a separate build as opposed to the rest of the samples?

Reply:
Please see the Neenu comment below